selenium.go

```diff
@@ -330,6 +363,44 @@ type WebDriver interface {
 	// ButtonUp causes the left mouse button to be released.
 	ButtonUp() error
 
+	// KeyPauseAction build a KeyAction which pauses for the supplied duration
+	KeyPauseAction(duration time.Duration) KeyAction
+	// KeyUpAction build a KeyAction press
+	KeyUpAction(key string) KeyAction
+	// KeyDownAction build a KeyAction which presses and holds the specified
+	// key until explicitly released by KeyUpAction or ReleaseActions
+	KeyDownAction(key string) KeyAction
+	// StoreKeyActions
+	//
+	// inputId: unique device identifier for the stored action
+	//
+	// actions: any number of KeyActions
+	StoreKeyActions(inputId string, actions ...KeyAction)
+
+	// PointerPause build a PointerAction which pauses for the supplied duration
+	PointerPauseAction(duration time.Duration) PointerAction
+	// PointerMove build a PointerAction which moves the pointer
+	PointerMoveAction(duration time.Duration, offset Point, origin PointerMoveOrigin) PointerAction
+	// PointerUp build an action which releases the specified pointer key
+	PointerUpAction(button MouseButton) PointerAction
+	// PointerDown build a PointerAction which presses and holds the specified
+	// pointer key until explicitly released by PointerUp or ReleaseActions
+	PointerDownAction(button MouseButton) PointerAction
+	// StorePointerActions
+	//
+	// inputId: unique device identifier for the stored action
+	//
+	// pointerType: type of pointer - accepts: MousePointer, TouchPointer, PenPointer
+	//
+	//actions: any number of PointerActions
+	StorePointerActions(inputId string, pointerType PointerType, actions ...PointerAction)
```

Comment:
1) inputID, as earlier.
2) Let's just call it "pointer" to reduce stutter.
3) What are valid inputIDs? How does one obtain a unique device identifier? Add this to the documentation.